[New Product] Vuetify (#2764)
captn3m0/endoflife.date: products/vuetify.md

---
title: Vuetify
category: framework
iconSlug: vuetify
permalink: /vuetify

> alternate_urls:
> -   /vuetify

Reviewer: Alternate URLs need to be different (they're aliases)

```suggestion
```

versionCommand: npm list vuetify

> releasePolicyLink: https://vuetifyjs.com/en/introduction/roadmap/#released

Reviewer: Adds a changelogTemplate:

```suggestion
releasePolicyLink: https://vuetifyjs.com/en/introduction/roadmap/#released
changelogTemplate: https://github.com/vuetifyjs/vuetify/releases/tag/v__LATEST__
```

> releasePolicyLink: https://vuetifyjs.com/en/introduction/roadmap/#released

Reviewer: The LTS link actually gives a better overview of all the supported releases, so using that makes sense perhaps?

```suggestion
releasePolicyLink: https://vuetifyjs.com/en/introduction/long-term-support/
```

activeSupportColumn: true
releaseDateColumn: true

auto:
-   npm: vuetify

> 

Reviewer: The [Jupyter/Python variants](https://repology.org/projects/?search=vuetify) don't seem to use the same versioning so ignoring them.

```suggestion

identifiers:
-   purl: pkg:npm/vuetify
```

releases:
-   releaseCycle: "3"
    support: true
    eol: false
    latest: "3.1.12"

>     lts: true

Reviewer: ```suggestion
    lts: false
```

    latestReleaseDate: 2023-03-28
    releaseDate: 2022-11-01

>     link: https://github.com/vuetifyjs/vuetify/releases/tag/v3.1.12

Reviewer: Use changelogTemplate instead.

```suggestion
```

-   releaseCycle: "2"

>     support: true

Reviewer: 2.6.0 was the last version which added any features to v2. Active Development has only been on v3 since.

```suggestion
    support: 2021-11-15
```

    eol: false
    latest: "2.6.14"

>     lts: true

Reviewer: This would become LTS with the 2.7.0 release, which hasn't arrived yet.

```suggestion
    lts: false
```

    latestReleaseDate: 2023-01-10
    releaseDate: 2019-07-23

>     link: https://github.com/vuetifyjs/vuetify/releases/tag/v2.6.14

Reviewer: Use changelogTemplate instead.

```suggestion
```

-   releaseCycle: "1.5"

>     support: 2020-07-31

Reviewer: The [v2 release](https://github.com/vuetifyjs/vuetify/releases/tag/v2.0.0) (Jun 2019) said the following:
>What will be happening to v1.5?
>-  No more features

```suggestion
    support: 2019-06-24
```

    eol: 2020-07-31
    latest: "1.5.23"

>     lts: true

Reviewer: LTS Start date as per https://vuetifyjs.com/en/introduction/long-term-support/

```suggestion
    lts: 2019-07-31
```

    latestReleaseDate: 2020-02-21
    releaseDate: 2019-02-05

>     link: https://github.com/vuetifyjs/vuetify/releases/tag/v1.5.23

Reviewer: Use changelogTemplate instead.

```suggestion
```

---

> [Vuetify](https://vuetifyjs.com/) is a no design skills required UI Library with 

> > beautifully handcrafted Vue Components.

Reviewer: Slight reword idea:

```suggestion
> [Vuetify](https://vuetifyjs.com/) is an open-source Vue Component Framework. It is a collection of consistently styled UI components with features such as dynamic themes, global defaults, application layouts, and more.
```

> [Vuetify release cycle](https://vuetifyjs.com/en/introduction/long-term-support/). 
> Vuetify provides long-term support to the last major release for 18 months
> for critical bugs and security vulnerabilities.

Reviewer: ```suggestion
Each major release is supported with critical and security fixes for [a minimum of 18 months](https://vuetifyjs.com/en/introduction/long-term-support/) from the date of release of the last minor version in a given cycle, which marks the start of LTS.
```

> We understand that many projects that utilize Vuetify have development cycles
> that prevent upgrading to the latest version. In order to provide developers
> and businesses peace of mind when adopting Vuetify, we commit to at minimu
> of 18 months for critical bugs and security vulnerabilities for the latest minor of the last major release.

Reviewer: We use [a neutral third-person tone](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#tone-and-text), so avoid using "we". In this case, all of this is covered above already, so dropping this para.

```suggestion
```
